src/helpers/validateConfig.js

```diff
@@ -82,6 +82,15 @@ const validateConfig = (config) => {
         // eslint-disable-next-line no-param-reassign
         config.thresholdForFutureMessageSeconds = 300
     }
+    if (config.ethereum === undefined) {
+        throw new MissingConfigError('ethereum')
+    }
+    if (config.ethereum.privateKey === undefined && config.ethereum.generateWallet === undefined) {
```

Comment:
For user experience, might be good to also check that both aren't enabled at the same time and throw an error if that is the acse.